specifications/api/swagger.yaml

```diff
@@ -2202,13 +2202,55 @@ components:
         minimum_utxo_value:
           <<: *amount
           description: |
-            The minimum ada / Lovelace quantity required for new transaction outputs.
+            The absolute minimum quantity of ada required for a new transaction
+            output created with the wallet.
+
+            In general, the ledger rules require that every transaction output
+            has a minimum quantity of ada. This minimum quantity is determined
+            by an era-specific function whose value increases as the number of
+            different assets increases, as the quantity of any individual asset
+            increases, as the length of the target address increases, and if a
+            datum hash is added.
+
+            Therefore, the value reported by this field should only be viewed
+            as an absolute minimum, and only applies to outputs that send ada
+            (and no other assets) to Shelley-era addresses. If an output
+            contains other assets, specifies a datum hash, or sends funds to a
+            Byron-era address, then the minimum value required by the ledger
+            (and the wallet) will be higher than the value reported by this
+            field.
```

Comment:
With
```haskell
txOutputMinimumAdaQuantity txConstraints
                 (AD.maxLengthAddressFor $ Proxy @ShelleyKey)
                 TokenMap.empty
```
this seems either incorrect or misleading. I think we should either
1. Replace `AD.maxLengthAddressFor` with `minLengthAddress`
2. Reword this paragraph 

Actually, the first option might make some sense? (Even if one could say the overestimated Coin value is at odds with the paragraph too, if we not only consider the wallet's API)

Reply:
Thanks @Anviking, I think you've raised a good point.

I'll have a think about what we should do here, and make a PR to either adjust the comment, or adjust the definition in the way that you suggest.

I think the intention of the comment is that "If you're sending ada (and no other assets) to a Shelley address, then sending this amount of ada _or more_ should _always_ work". So in this sense, it's the minimum value that will _always_ be fine for _all_ Shelley addresses.

If we were to alter the definition to use `minLengthAddress`, but then the user specifies a Shelley address that is longer, then their payment would be rejected (if they had chosen to pay the value returned by this field). I would argue that this would make the `min_utxo_value` field even less useful that it already is.

As you know, I think we should mark this field as "deprecated" as soon as possible, since it's quite hard for the user to interpret the meaning (and of course, this protocol parameter no longer exists in the Alonzo and Babbage eras).

Reply:
@Anviking I've added a reminder to address this comment here: https://input-output.atlassian.net/browse/ADP-2039.

Reply:
> If you're sending ada (and no other assets) to a Shelley address, then sending this amount of ada or more should always work when creating transactions with the wallet

I think that's fair, but this is what should be said in the docs then. So e.g. something more like:

> In Babbage the minimum, even for ada-only outputs, is not constant. It depends upon the serialised length of the output, including address and ada quantity. What is returned by this field is a value which works for sending ada to any Shelley address. The true minimum may be marginally smaller with short addresses, or bigger with longer Byron addresses or included tokens. The field may be deprecated or removed in future versions of the wallet.

> I think we should mark this field as "deprecated" as soon as possible, since it's quite hard for the user to interpret the meaning

I do agree 💯